frontend/src/components/NewRunParametersV2.tsx

```diff
@@ -71,6 +71,11 @@ function convertInput(paramStr: string, paramType: ParameterType_ParameterTypeEn
         return Number(paramStr);
       }
       return null;
+    case ParameterType_ParameterTypeEnum.NUMBER_DOUBLE:
+      if (!Number.isNaN(Number(paramStr))) {
```

Comment:
NIT: Is it necessary to use `Number()`? Can we simplify it by using `!Number.isNaN(paramStr)`?

Reply:
Yes, I think so. The input of isNaN() should be number.

Reply:
How about `!isNaN(paramStr)`?  If we can use the standard library to do conversion for us, it is preferred: https://www.w3schools.com/jsref/jsref_isnan.asp